data_tfms.ipynb

```diff
@@ -39,7 +39,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">train_df.to_json("data/raw/train.json", orient="records")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>PR it when you are updating paths? PLIJ</p><p> </p>

Reply:
Resolved by 6885478